From: Boyuan Yang <byang@debian.org>
Date: Tue, 13 Jun 2023 10:02:26 -0400
Subject: Restore cairocffi/VERSION

Otherwise sphinx would report the following error:

python3 -m sphinx -N -bhtml docs/ docs/_build/html
Running Sphinx v5.3.0

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/sphinx/config.py", line 350, in eval_config_file
    exec(code, namespace)
  File "/<<PKGBUILDDIR>>/docs/conf.py", line 9, in <module>
    Path(__file__).parent.parent / 'cairocffi' / 'VERSION').read_text().strip()
                                                            ^^^^^^^^^^^
  File "/usr/lib/python3.11/pathlib.py", line 1059, in read_text
    with self.open(mode='r', encoding=encoding, errors=errors) as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/pathlib.py", line 1045, in open
    return io.open(self, mode, buffering, encoding, errors, newline)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/<<PKGBUILDDIR>>/cairocffi/VERSION'
---
 cairocffi/VERSION | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 cairocffi/VERSION

diff --git a/cairocffi/VERSION b/cairocffi/VERSION
new file mode 100644
index 0000000..9c6d629
--- /dev/null
+++ b/cairocffi/VERSION
@@ -0,0 +1 @@
+1.7.1
